docs/release-notes.md

```diff
@@ -42,20 +57,20 @@
 
     The webserver configurations have been added to harden access permissions.
 
-- **DietPi-Software**  :octicons-commit-24: **Docker** :octicons-arrow-right-16: On fresh installs or reinstalls, logs are now done with limited verbosity to systemd-journald (RAM), accessible via `"journalctl -u docker -u containerd"` to reduce disk writes.
-
-    Docker can be reinstalled via `"dietpi-software reinstall 162"`.
-
-    Many thanks to @SaturnusDJ for doing this suggestion: [Issue #2388](https://github.com/MichaIng/DietPi/issues/2388)
-
 - **DietPi-Software**  :octicons-commit-24: **Mosquitto** :octicons-arrow-right-16: The official APT repository is now used where possible, which currently excludes ARMv8/arm64 and Raspbian/Debian Bullseye. This change is applied via reinstall during DietPi update.
 
     Many thanks to @marcobrianza for doing this suggestion: [Issue #3042](https://github.com/MichaIng/DietPi/issues/3042)
 
-- **DietPi-Software**  :octicons-commit-24: **Cuberite** :octicons-arrow-right-16: A wrong directory name based on a typo has been fixed.
+- **DietPi-Software** :octicons-commit-24: **Cuberite** :octicons-arrow-right-16: A wrong directory name based on a typo has been fixed.
 
     A reinstall updates Cuberite and moves its install directory to `/mnt/dietpi_userdata/cuberite` as intended. This is applied via DietPi update but a backup is created to cover issues due to potential structural changes, especially for older instances installed at /etc/cuberite.
 
+### Documentation improvements
+
+- **Installation guide** :octicons-arrow-right-16: It has been fully changed & simplified. There is a single page with more tabs. The page highlights different steps used for different platforms, while it keeps the common structure. Many thanks to @StephanStS for doing these updates.
+
+- **Writing rules** :octicons-arrow-right-16: The documentation has evolved. It starts to have a file naming convention, more url updated and so on. Many thanks to @StephanStS for doing these updates.   
+
 ### APT Changes
```

Comment:
Ah there was a typo in DietPi upstream changelog as well, it should be "API Changes" 😉.